src/test/kotlin/com/exactpro/th2/check1/rule/RuleFactoryTest.kt

```diff
@@ -65,10 +64,13 @@ class RuleFactoryTest {
                 .setParentEventId(EventID.newBuilder().setId("root").build())
                 .setCheckpoint(Checkpoint.newBuilder().setId(EventUtils.generateUUID()).build()).build()
 
-        assertThrows<RuleCreationException> {
+        val mainException = assertThrows<RuleCreationException> {
             ruleFactory.createCheckRule(request, true)
         }
 
+        assertEquals("An error occurred while creating rule", mainException.message)
+        assertEquals("Session alias cannot be empty", mainException.cause?.message)
```

Comment:
You do the same things here and in `failed rule creation with missed checkpoint and invalid chain id` test.
You can extract this into a method and reuse part of the code